.github/workflows/builder.yml

```diff
@@ -20,6 +20,9 @@ jobs:
       run: |
         brew install sdl2 openal-soft sdl2_mixer libpng libvorbis sccache googletest
         brew reinstall libpng
+        brew --cellar libpng
+        brew list --versions libpng
+        brew list --versions libpng | egrep -o '[0-9]+\.[0-9]+\.[0-9]+'
```

Comment:
These don't hurt, but now the pointless reinstall can be removed.

Reply:
Removed